service/src/main/resources/WEB-INF/templates/assistance-need/decision_sv.properties

```diff
@@ -53,6 +53,8 @@ statuses.ACCEPTED=godkänt
 statuses.REJECTED=avvisat
 decisionStatus=Beslutet är
 decisionMadeOn=Beslut fattat den
+legalInstructions=Tillämpade bestämmelser
+legalInstructionsText=Lag om småbarnspedagogik, 3 a kap
 jurisdiction=Befogenhet
 jurisdictionText=Delegationsbeslut om beslutanderätten för tjänsteinnehavarna av personalen vid den finskspråkiga småbarnspedagogiken och tillväxt- och lärandegrenen A del 3 § 3 pkt
```

Comment:
`jurisdictionText`, `startDate`, `unitMayChange`, `structuralMotivationOptions.additionalStaff` need to be updated as well

Reply:
Good catch!